### ISSUE TITLE
Problema con comandos

### ISSUE THREAD

**Issue by LuckserSky34** [Ajneb97/PlayerKits2#18]
Tengo otro plugin que también usa el comando /kit y esto hace que el plugin se deshabilite sin que funcione el comando principal ni
tampoco los aliases, en la versión 1.20.4 no pasaba solo pasa en la 1.21

**Comment by Ajneb97** [comment]
Dices que el plugin se deshabilita estando en rojo al usar el comando /plugins ? Si es asi, mandame el error que te aparece en la consola cuando el plugin se intenta activar.

**Comment by LuckserSky34** [comment]
no, me refiero que cuando otro plugin utiliza el mismo comando /kit el plugin playerkits deja de funcionar completamente. sin poder usar otros comandos o recargar el propio plugin, y que en versiones anteriores esto no ocurría.

**Comment by Ajneb97** [comment]
Tienes que deshabilitar el comando kit en el otro plugin, si no intenta con /kits o /playerkits

**Comment by LuckserSky34** [comment]
por eso digo que aunque intente usar /kits o /playerkits no funcionan y esto antes no me pasaba, la unica solucion que encontre es hacer que el playerkits se cargue antes que mi otro plugin, pero si le hago reload al plugin deja de funcionar

**Comment by Ajneb97** [comment]
Si no te funciona /playerkits entonces el otro plugin tiene prioridad y las maneras de solucionarlo son; borrar el otro plugin o modificar el comando en el archivo plugin.yml dentro del .jar
